content/en/admin/config.md

```diff
@@ -22,6 +22,24 @@ For convenience, it can read them from a flat file called `.env.production` in t
 * `LOCAL_DOMAIN`
 * `WEB_DOMAIN`
 * `ALTERNATE_DOMAINS`
+* `AUTHORIZED_FETCH`
```

Comment:
If these become subheaders instead they'll be linkable

Reply:
should we change… all of them, then? :/

Reply:
Just the ones that have content, in my opinion (these 2 new ones)

Reply:
:woman_shrugging: